scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -19,6 +19,29 @@ if String.IsNullOrEmpty githubEventPath then
 
     Environment.Exit 2
 
+let accessToken = Environment.GetEnvironmentVariable "ACCESS_TOKEN"
+(*
+To save your GitHub access token as an environment variable in GitHub
+Actions, you can use the secrets feature in GitHub. In your GitHub Actions
+workflow file, you can reference the secret as an environment variable using
+the syntax ${{ secrets.GITHUB_TOKEN }}. For example:
+
+```
+name: My Workflow
+
+on: [push, pull_request]
+
+jobs:
+  my_job:
+    runs-on: ubuntu-latest
+    env:
+      ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
```

Comment:
Is this part needed to be in the example? Apart from this, it LGTM.

Reply:
Removed it